README.md

```diff
@@ -25,6 +25,8 @@ If you want to disable pylint, you can [disable this extension](https://code.vis
 | pylint.importStrategy   | `useBundled`                                                                                                                           | Setting to choose where to load `pylint` from. `useBundled` picks pylint bundled with the extension. `fromEnvironment` uses `pylint` available in the environment.                                                                                                                                                       |
 | pylint.showNotification | `off`                                                                                                                                  | Setting to control when a notification is shown.                                                                                                                                                                                                                                                                         |
 | pylint.lintOnChange     | `false`                                                                                                                                | (experimental) Setting to control linting on change feature.                                                                                                                                                                                                                                                             |
+| pylint.ignorePatterns   | `[]`                                                                                                                                   | Glob patterns to exclude files and directories while linting.                                                                                                                                                                                                                                                            |
+| pylint.includeStdLib    | `false`                                                                                                                                | Setting to control whether to include standard library paths while linting.                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| pylint.includeStdLib    | `false`                                                                                                                                | Controls whether to perform linting on Python's standard library files or directories.                                                                                                                                                                                                                                        |
```

Reply:
I'm wondering if we should add a note about how this relates to ignorePatterns ? E.g. if someone adds some stdlib file to ignorePatterns   but this  includeStdLib   is set to true, which one takes precedence? 
(we could add this scenario to our docs only instead of the setting description too)

Reply:
It should likely go in the docs. StdLib check occurs before pattern check.